src/animated/Controller.ts

```diff
@@ -304,56 +337,52 @@ class Controller<P extends any = {}> {
             parent.setValue(isInterpolation ? toValue : value, false)
           }
 
-          return {
-            ...acc,
-            [name]: {
-              ...entry,
-              name,
-              parent,
-              interpolation,
-              animatedValues,
-              toValues,
-              previous: newValue,
-              config: toConfig,
-              fromValues: toArray(parent.getValue()),
-              immediate: callProp(immediate, name),
-              initialVelocity: withDefault(toConfig.velocity, 0),
-              clamp: withDefault(toConfig.clamp, false),
-              precision: withDefault(toConfig.precision, 0.01),
-              tension: withDefault(toConfig.tension, 170),
-              friction: withDefault(toConfig.friction, 26),
-              mass: withDefault(toConfig.mass, 1),
-              duration: toConfig.duration,
-              easing: withDefault(toConfig.easing, (t: number) => t),
-              decay: toConfig.decay,
-            },
+          changed = true
+          acc[name] = {
```

Comment:
I think it's better to avoid creating a new copy of `acc` for every changed animation, so I switched this to a simple assignment. WDYT?

Reply:
looks good!